src/main/webapp/i18n/de/exam.json

```
@@ -338,7 +338,9 @@
                 "noActionRequired": "Artemis erfordert keine weitere Aktion, das Fenster kann geschlossen werden.",
                 "followExamProtocol": "Halte dich an die Klausuranweisungen deiner Lehrenden.",
                 "button": "Zusammenfassung der Klausur anzeigen{{ countdown }}"
-            }
+            },
+            "saveExercise": "Aufgabe Speichern",
+            "exerciseSaved": "Aufgabe Gespeichert"
```

```suggestion
            "saveExercise": "Aufgabe speichern",
            "exerciseSaved": "Aufgabe gespeichert"
```
I think we don't use title case in buttons

---

I think we should define a specific rule for the buttons for future implementations and document it, because Artemis is currently really inconsistent in buttons. I am changing this to lower case now 👍